integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -109,13 +98,10 @@ public void remove_key(String key) throws IOException {
     @When("user sends a valid request to authorization end point")
     public void user_sends_a_valid_request_to_authorization_end_point()
             throws IOException, InterruptedException, URISyntaxException {
-        System.out.println("DEV_AUTHORIZATION_URI is --------" + DEV_AUTHORIZATION_URI);
+        System.out.println("DEV_AUTHORIZATION_URI is --------" + devAuthorizationUri);
```

Comment:
Are we not using any logging framework, if not, can we default to the internal Java logger for log statements please? They provide detailed pre-log clarity such as timestamps and caller details.